elements/lawrencium.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Lawrencium
+---
+
+Symbol : Lr
+
+Atomic Number : 103cd
```

Comment:
Did you mean to leave this cd here? Other than that, this seems great!

Reply:
Edited with new version! Thank you